rpcd/playbooks/roles/rpc_maas/tasks/raxmon_agent_install.yml

```diff
@@ -38,6 +38,11 @@
     group: "root"
     mode: "0600"
 
+- name: Verify raxmon-entities-list and MaaS API authentication does not fail
+  shell: "raxmon-entities-list"
+  register: raxmon_output
+  failed_when: raxmon_output.stderr != ""
```

Comment:
raxmon-entities-list exists 0 on success and 1 on failure so you could check raxmon_output.rc instead of stderr.